### ISSUE TITLE
[Bug Report][3.1.3] Unable to get access to the type Anchor from 'vuetify/lib/components'

### ISSUE THREAD

**Issue by moeezsaiyam** [vuetifyjs/vuetify#16559]
### Environment
**Vuetify Version:** 3.1.3
**Vue Version:** 3.2.45
**Browsers:** Chrome 108.0.0.0
**OS:** Mac OS 10.15.7

### Steps to reproduce
Open in VS Code or run 'npm run typecheck'. Will get the following error: Cannot find module 'vuetify/lib/components' or its corresponding type declarations. Or if import from 'vuetify/components' will get the following error: Module '"vuetify/components"' declares 'Anchor' locally, but it is not exported.

### Expected Behavior
Type Anchor should be imported

### Actual Behavior
Get Type error while import

### Reproduction Link
[https://github.com/simplycubed/firebase-vue-starter/blob/main/packages/dashboard/src/layouts/components/UserProfile.vue](https://github.com/simplycubed/firebase-vue-starter/blob/main/packages/dashboard/src/layouts/components/UserProfile.vue)

**Comment by KaelWD** [comment]
Duplicate of #16680

**Comment by matxx** [comment]
it is still not exposed, as of vuetify@3.7.18 (even after #16680 resolved)

[example](https://github.com/orgs/community/discussions/154054) : 

```vue
<template>
  <v-tooltip :location="tooltipLocation">
     ...things...
  </v-tooltip>
</template>
<script setup lang="ts">
const props = withDefaults(
  defineProps<{
    tooltipLocation?: Anchor
  }>(),
  {
    tooltipLocation: 'top',
  },
)
```

`import type { Anchor } from 'vuetify'`
gives
> Module '"vuetify"' has no exported member 'Anchor'.ts(2305)

trying `import type { Anchor } from 'vuetify/lib/util/anchor'`
gives
> Cannot find module 'vuetify/lib/util/anchor' or its corresponding type declarations.ts(2307)

**Comment by matxx** [comment]
ah sorry 😅 

just tested with `v3.8.0-beta.0` and it indeed works
